packages/project-types/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@sap-ux/project-types",
```

Comment:
No, it provides `enum` values that can't be imported as `devDependency`

Reply:
If the project does more than types then I find the project name misleading and we need to discuss it again.
Btw, is there any `enum` that you cannot export as `const enum`? Wouldn't this solve the problem?

Reply:
I did not find any way to do an `import type` of an `enum` or `const enum` and use it's value. If you find a way I'm very happy to use it. 
If we do not find a way and want to use the module as pure type only, we need to remove the enums.

Reply:
My understanding is that you need a `const enum`, still need to do `import` (without type) but you can have the module as `devDependency` because during the transpilation the enums are inlined.

This is a topic for which we need the knowledge of @ullasholla

Reply:
> My understanding is that you need a const enum, still need to do import (without type) but you can have the module as devDependency because during the transpilation the enums are inlined.

This is my understanding as well. You may have to play around with the bundling to check if it actually works as we expect it.

If we do stick with const enums, does this mean that we'll need ban non-const enums in `project-types`? Maybe using something like this in `project-types`: https://github.com/typescript-eslint/typescript-eslint/issues/561#issuecomment-593059472

Reply:
I like the idea of adding that rule here.

Reply:
**It works!** thanks @tobiasqueck  and @ullasholla 

_Here is what I've done_
Changed to `const enum` in module `@sap-ux/project-types`  
-> build `@sap-ux/project-types`

Changed `@sap-ux/project-access` to consume `@sap-ux/project-types` as `devDependecy`, then imported enum with

```
import { FileName } from '@sap-ux/project-types';
..
const manifest = await readJSON<Manifest>(join(manifestPath, FileName.Manifest));
```

-> build and check in `dist`, the const enum values have been replaced with string, e.g.

```
const manifest = yield file_1.readJSON(path_1.join(manifestPath, "manifest.json" /* Manifest */));
```

Now I use this const enum value from `@sap-ux/project-types` in other module like `@sap/ux-project-access`, which is bundled in extension.  
-> add code to output on error console

Build vsix

Install and run in VSCode Insiders
-> error console shows
```
ERR [Extension Host] JustToProof.Works: 'thisworks'
ERR [Extension Host] JustToProof.MakeSure: 'thisstillworks'
```

I've also added eslint rule `no-restricted-syntax`